src/main/frontend/handler/web/nfs.cljs

```diff
@@ -86,6 +87,22 @@
                        (keyword (util/get-file-ext (:file/path file)))))
           files))
 
+(defn- precheck-graph-dir
+  [dir files]
+  (when (= (string/lower-case (path/basename dir))
+           "logseq")
+    (state/pub-event!
+     [:notification/show {:content [:div "The folder name "
+                                    [:code "logseq"]
+                                    " is not suitable for a graph name. Please unlink this graph and choose a different name."]
+                          :status :warning
+                          :clear?  false}]))
+  (when (seq (filter #(string/ends-with? (:path %) "/logseq/config.edn") files))
```

Comment:
`filter` will go through all the collection items (Although it is lazy, it will not have any impact on performance.)
`some` is more suitable here

Reply:
Nice catch